README.md

```diff
@@ -24,5 +24,10 @@ and execute it:
 
 ## Dependencies
 
-ElasticSearch provides the search functionality, and must be running at the $ELASTIC_SEARCH_ENDPOINT listed in the environment variables.
+```bash
```

Comment:
Changed. The original was simply from the CentOS dev env - which is overridden at app startup by create_dev_env setup script.